src/main/steps/respondentSequence.ts

```diff
@@ -89,7 +89,6 @@ const sequence: Step[] = [
 
 export const respondentSequence = ((): Step[] => {
   return sequence.map(step => ({
-    ...step,
```

Comment:
do we not need the other step properties?

Reply:
This ...step was added when our steps looked like this:
  {
    url: DETAILS_OTHER_PROCEEDINGS,
    showInSection: Sections.OtherCourtCases,
    getNextStep: () => HOW_THE_COURTS_WILL_CONTACT_YOU,
  },
but now we don't have showInSection we can remove it.